from-dump/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM quetzacoalt/docker-internal-images:1.7
+
+ENV PS_DOMAIN localhost
+ENV DUMP_PHP_FILES prestashop-dataset
+ENV DUMP_SQL_FILE prestashop-dataset.sql
+
+# Copy files
+COPY $DUMP_PHP_FILES/ /var/www/html/
+RUN ln -s /var/www/html/ /var/www/html/$DUMP_PHP_FILES/
```

Comment:
I updated the Dockerfile for the database. The URL is now set on the root folder.